publicAPI/crest_utils.py

```diff
@@ -247,12 +248,19 @@ def fetch_crest_endpoint(
         'User-Agent': config.get('GLOBAL', 'useragent')
     }
 
-    # no try-except, catch in caller
-    # done to make logging path easier
-    req = requests.get(
-        crest_url,
-        headers=headers
-    )
+    @retry(wait_fixed=2000, stop_max_delay=10000)
+    def fetch_crest_endpoint_get():
+        # no try-except, catch in caller
+        # done to make logging path easier
+        req = requests.get(
+            crest_url,
+            headers=headers
+        )
+        if req.status_code == 502:          # Server Error: Bad Gateway for url
```

Comment:
unit_tests should cover us for badly-formed requests, and weekly cron will fail if ESI API updates.  Will still raise (same failed request) log and alert if things go wrong